src/LibHdb++Cassandra.cpp

```diff
@@ -1069,9 +1235,9 @@ void HdbPPCassandra::extract_and_bind_rw_values(CassStatement* statement,
 					extract_and_bind_encoded(statement,param_index, data_format, data, EXTRACT_READ); break;
 				default:
 				{
-					TangoSys_MemStream os;
-					os << "Attribute " << data->attr_name << " type (" << (int)(data_type) << ")) not supported";
-					cerr << __func__ << ": " << os.str() << endl;
+					// TODO (SJ) Handle error state how?
+					DEBUG_MSG("Attribute " << data->attr_name << " type (" << (int)(data_type) << ")) not supported" << endl);
```

Comment:
I think it would be good to find a way to throw an exception up to the user/archiver in this case because it seems we are receiving events from an unsupported type.